arc-api/server/models/subscription.js

```diff
@@ -0,0 +1,272 @@
+var request = require('request');
+var KeyStore = require('./key-store');
+var debug = require('debug')('strong-arc:subscription');
+
+// Extract the access token from the request
+function getAccessToken(req) {
+  var token = req.query.access_token;
+  if (token) {
+    return token;
+  }
+  token = req.get('authorization');
+  if (typeof token === 'string') {
+    // Add support for oAuth 2.0 bearer token
+    // http://tools.ietf.org/html/rfc6750
+    if (token.indexOf('Bearer ') === 0) {
```

Comment:
I was trying to use the same code from AccessToken but this function is not exported.